### DISCUSSION TITLE
WinUtil.exe even with Exceptions popup says Trojan:Win32AgentTesla!ml and Sig OK - What to DO??

### DISCUSSION THREAD

**Discussion by AdenaWilder** [ChrisTitusTech/winutil#891] [Q&A]
https://github.com/damienvanrobaeys/Windows_Sandbox_Editor/issues/1
![WinUtil Digital Signature Verified and still a problem](https://github.com/ChrisTitusTech/winutil/assets/53023792/10866824-5f78-4a64-8aed-8b41ca966186)

![image](https://github.com/ChrisTitusTech/winutil/assets/53023792/7372bf58-11d8-404f-91c7-db6e2d74c64a)

![winutil popup more info Trojan Win32 AgentTesla](https://github.com/ChrisTitusTech/winutil/assets/53023792/08933ddd-84a9-4719-9e23-d86ef0b330b4)
![winutil popup even with exceptions in Windows Security](https://github.com/ChrisTitusTech/winutil/assets/53023792/d9ea6d76-e27f-4370-83a7-689a34438033)

**Comment by anonyao** [comment]
Haha lol first of all dont use win defender antivirus , use eset internet security . And u wont have those false alarm agent its what they warn becuse they dont want u to debloat

**Reply by Davy49** [reply]
Hi,
Your comment is interesting, windows security is included for free with the windows operating system at no additional cost. You mentioned eset internet security, how much does that software cost as I feel pretty sure that it's not free.

**Reply by anonyao** [reply]
Its free premium just change mail every 30days in eset and done

**Reply by Davy49** [reply]
anonyao,
Isn't that kind of a hassle having to change email address every 30 days ?

**Reply by AdenaWilder** [reply]
Ok - so just ignore ??? Seems like something Titus might want to address - or give a response to ??

**Reply by AdenaWilder** [reply]
A little irritated by this since I paid $10 (?)ish for this “great app” ?!
